spring-cloud-gcp-samples/spring-cloud-gcp-data-datastore-basic-sample/pom.xml

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="UTF-8"?>
+ <?xml version="1.0" encoding="UTF-8"?>
```

Comment:
??

Reply:
I think this must've been the auto-formating

Reply:
fixed :)